scalapb-runtime-grpc/src/main/scala/scalapb/grpc/ProtoInputStream.scala

```diff
@@ -0,0 +1,66 @@
+package scalapb.grpc
+
+import com.google.protobuf.CodedOutputStream
+import scalapb.GeneratedMessage
+
+import java.io.{ByteArrayInputStream, InputStream}
+
+/**
+  * Allows skipping serialization completely when the io.grpc.inprocess.InProcessTransport is used.
+  * Inspired by https://github.com/grpc/grpc-java/blob/master/protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoInputStream.java
+  */
+class ProtoInputStream[T <: GeneratedMessage](msg: T, val marshaller: Marshaller[T]) extends InputStream {
```

Comment:
Can you add unit tests for this class?

Reply:
added in https://github.com/scalapb/ScalaPB/pull/1615/commits/eaec5ad36bf236c6993a999df6ab82e37948cebb